tools/buildsys/src/builder.rs

```diff
@@ -358,6 +443,8 @@ fn docker(args: &[String], retry: Retry) -> Result<Output> {
         retry_messages = messages;
     }
 
+    println!("Running command:\ndocker {}", args.join(" /\n"));
```

Comment:
This is only one place out of two we `println!()` in this file. Is that intentional? I think so but wanted to ensure this isn't a hold over from debugging and this output is doing something useful for logs/output for the user?

Reply:
> This is only one place out of two we println!() in this file. Is that intentional? I think so but wanted to ensure this isn't a hold over from debugging and this output is doing something useful for logs/output for the user?

Good catch. Fixed.